grease-cli/src/Grease/Main.hs

```diff
@@ -429,9 +429,10 @@ interestingConcretizedShapes ::
 interestingConcretizedShapes names initArgs (ConcArgs cArgs) =
   let cShapes = fmapFC (Shape.tagWithType . concShape) cArgs in
   let initArgs' = fmapFC Shape.tagWithType initArgs in
+  let isLlvmArg name = "%" `List.isPrefixOf` name in
   Ctx.zipWith
     (\(Const name) (Const isDefault) ->
-      Const (name `List.elem` interestingRegs && not isDefault))
+      Const ((name `List.elem` interestingRegs || isLlvmArg name) && not isDefault))
```

Comment:
We have now changed the meaning of what it means for an argument to be "interesting", so please update the relevant part of the Haddocks above to reflect this.